matRad_sampling.m

```diff
@@ -141,7 +146,7 @@
       FlagParforProgressDisp = false;
    end
   
-   for i = 1:pln.multScen.totNumScen
+   parfor i = 1:pln.multScen.totNumScen
```

Comment:
Without the Parallel computing toolbox, the parfor may be marginally slower than a normal for loop . 
As we arent focussing that much on speed this works out just fine , however for future we could have a logic variable to check if Parallel computing tool box is present and then choose which loop to use.

Reply:
isn't this whole codeblock in such an if/else statement depending on available license? Maybe it would even make more sense the other way round, trying to minimize double code as much as possible (i.e., kind of externalizing the inner loop into a function, such that this code is not doubled)